upspeak/index.html

```diff
@@ -0,0 +1,315 @@
+    <div class="row">
+        {{#profiles}}
+        <div class="col-md-3 col-lg-3 mb-5">
+            <div class="px-4">
+                <div class="hover-profile-card" onclick="openMentorProfile('{{index}}')">
+                    <img src="images/{{image}}"
+                         class="rounded-circle img img-center img-fluid shadow shadow-lg--hover imgHover"
+                    >
+                </div>
+                <div class="pt-4 text-center">
+                    <h5 class="title">
+                        <span class="d-block mb-1">{{name}}</span>
+                    </h5>
+                    <span class="text-light mentor-title">{{title}}</span>
+                </div>
+            </div>
+        </div>
+        {{/profiles}}
+    </div>
+</script>
+
+<script type="text/html" id="template-profile-modal-content">
+    <div class="text-center">
+        <img src="images/{{image}}" alt="Rounded image"
+             class="img-fluid rounded-circle profile-image border img-center shadow mb-4">
+        <h4>{{name}}</h4>
+        <span>{{title}}</span>
+        <div class="mt-3">
+            {{#socialLinks}}
+            <a href="{{url}}"
+               target="_blank"
+               class="btn btn-{{platform}} btn-icon-only rounded-circle btn-icon-rounded">
+                <i class="{{icon}}"></i>
+            </a>
+            {{/socialLinks}}
+        </div>
+    </div>
+    <hr/>
+    <p class="text-justify">{{{bio}}}</p>
+    <hr/>
+    <h5>Areas of expertise :</h5>
+    <ul>
+        {{#areasOfExpertise}}
+        <li>
+            {{.}}
+        </li>
+        {{/areasOfExpertise}}
+    </ul>
+    <hr/>
+    <h5>Projects :</h5>
+    <ul>
+        {{#projects}}
+        <li>
+            {{.}}
+        </li>
+        {{/projects}}
+    </ul>
+    <hr/>
+</script>
+
+<script type="text/html" id="template-featured-stories">
+    <div class="row">
+        {{#data}}
+        <div class="col-md-4 mb-5">
+            <a href="{{url}}" target="_blank">
+                <div class="card shadow featured-story mb-4">
+                    <img src="images/featured-stories/{{image}}" alt="Rounded image"
+                         class="img-fluid rounded shadow img-full">
+                    <div class="card-body">
+                        <h6>{{title}}</h6>
+                    </div>
+                    <div class="card-footer">
+                        <div class="author">
+                            <img src="images/featured-stories/{{author.image}}" alt="..."
+                                 class="avatar img-raised">
+                            <span>{{author.name}}</span>
+                        </div>
+                        <div class="stats ml-auto">
+                            <i class="far fa-clock"></i> {{readingTime}} min read
+                        </div>
+                    </div>
+                </div>
+            </a>
+        </div>
+        {{/data}}
+    </div>
+</script>
```

Comment:
Good job @gajindubandara! I guess we can remove this part for now, since we don't have a profile section ATM.

Reply:
Yeah sure!